scripts/cache_warmer/start.py

```diff
@@ -0,0 +1,47 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv("../../.env")
+TX_SERVICE_URL = os.getenv("TRANSACTION_SERVICE_URL")
+PRINT_FORMAT = "{0:<10} {1:>8}::{2:>8}"
+
+
+def get_base_url() -> str:
+    if "staging" not in TX_SERVICE_URL:
+        return "https://safe-client.rinkeby.gnosis.io" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client.mainnet.gnosis.io"
+    else:
+        return "https://safe-client-rinkeby.staging.gnosisdev.com" if "rinkeby" in TX_SERVICE_URL \
+            else "https://safe-client-mainnet.staging.gnosisdev.com"
+
+
+def load_safes() -> list[str]:
```

Comment:
If you use `list[str]` it will require Python 3.9. If you want to support Python 3.8 (more extended), you should better use `List[str]` (and `List` should be imported from `typing`)